components/common/typescript.gni

```diff
@@ -137,6 +137,17 @@ template("transpile_web_ui") {
       "--depfile_path=" + rebase_path(depfile),
     ]
 
+    # Trezor bridge uses trezor-connect library that needs to be loaded
+    # with browser alias to open popup window and handle device communication
+    # via WebHID
+    if (resource_name == "trezor_bridge") {
+      args += [ "--webpack_alias=browser" ]
+    } else {
+      # For explanation of "chromeapp", see:
+      # https://github.com/brave/brave-browser/issues/5587
+      args += [ "--webpack_alias=chromeapp" ]
+    }
```

Comment:
This is great but IMO just 1 more level please - let's have the invoker be able to pass through `alias_fields`. Since the invoker should define whether the JS is targeted for browser or "chromeapp".

So trezor_bridge GN would specify `alias_fields = [ "browser" ]` and here in the template we would pass it through to python. It's ok to keep the default as `"chromeapp"` as you've already done.

Also, try to pass through an array vs a single string, for cases where we may need to provide multiple and won't have to handle string or array at that time.

Reply:
array can be follow-up, as that can be a bit more difficult to pass and I know this is an important fix

Reply:
done, thanks